index.bs

```diff
@@ -1566,12 +1566,35 @@ for each [=sensor types=]:
     especially when doing so would not make sense.
 -   A set of [=identifying parameters=]. TODO: replace that by an abstract operation.
 
-
 <h3 id="permission-api">Extending the Permission API</h3>
 
-Provide guidance on how to extend the Permission API [[PERMISSIONS]]
-for each [=sensor types=].
+{{Sensor}} interface for concrete [=sensor=] must protect it's [=sensor reading|reading=]
+by associated {{PermissionName}} or more complex {{PermissionDescriptor}}.
+[=Low-level=] {{Sensor|sensor}} may use it's interface name as a {{PermissionName}}, for instance
+"gyroscope" or "accelerometer". [=sensor fusion|Fusion sensors=] must
+[=request permission to use|request permission to use=] sensors used as a source of fusion.
+
+Even though, it might be difficult to reconstruct [=low-level=] [=sensor readings=] from
+fused data, some of the original information might be inferred. For example, it is easy to
+deduce user's orientation in space if absolute or geomagnetic orientation sensors are used,
+therefore, those sensors must [=request permission to use|request permission to use=]
+"magnetometer" as it provides information about orientation of device in relation to Earth's
+magnetic field. In contrast, relative orientation sensor does not expose such information, thus,
+does not need to [=request permission to use|request permission to use=] "magnetometer".
```

Comment:
it does

Reply:
Brainstorming some examples that I don't think (subjectively) require permission: 

- ambient light
- ambient temperature
- barometric pressure

I'm interested in reasonable scenarios that counter my initial intuition

Reply:
@pozdnyakov you mean relative orientation sensor readings fused from gyro and accel need magnetometer, or comment was for different line? 😄 

@rwaldron ALS needs permission, there are pin skimming research papers. Ambient temperature and pressure, I'm not sure, maybe you are right. Do you want me to add exception for such sensors?

Reply:
Assume someone lives alone at sea level (1 Bar) on the east coast of America and that room temperature inside their home is 20°C. If an ambient temperature sensor says it's 5°C this person is more like to be outdoor in winter than in their home. If a barometric pressure sensor says it's 0.5 Bar this person is more likely to be at the top of a 5000 meter mountain than in their home. In either case it's possible to deduce that the person is not at home and to safely burgle their home without being disturbed.

Reply:
@fivdi Thanks, those are possible attack vectors for information leaks.

@rwaldron I thought about possible scenarios where permission might not be required and here are few things to consider:

- UA can decide to autogrant permission to access 'safe' sensors.
- Attack vectors are unknown, would be better to have permission token, so that UA might patch and remove autogranting or apply other mitigation strategy for previously 'safe' sensors.
- GS API spec and all concrete sensor derivatives rely on Permission API integration and need permission tokens. Also we don't have sensors that fall into 'no permission token needed' category at this moment.

My proposal is to keep Permission API extendability for new sensors as is.

I can create issue for Level2 to investigate possibility of supporting sensors without dedicated permission token / descriptor.

Reply:
1 just meant that it would be better to add "it" => "It does not",  but I see now how it actually looks :/ sorry for the hassle..

Reply:
@pozdnyakov fixed. Thanks.

Reply:
@rwaldron Rick, could you please take a look again? Thanks!